mongonaut/sites.py

```diff
@@ -5,8 +5,6 @@
 
 
 class BaseMongoAdmin(object):
-    """Base model class for replication django.site.ModelAdmin class.
```

Comment:
Keep this comment, and all the other class/method/function comments that have been removed. It makes it harder for others coming into the code wen there are fewer comments explaining, how/why/what something does or exists.